src/delete.rs

```diff
@@ -0,0 +1,26 @@
+use crate::{error::Result, query::Query, sql::Bind, Client};
+
+#[must_use]
+#[derive(Clone)]
+pub struct Delete {
+    query: Query,
+}
+
+impl Delete {
+    pub(crate) fn new(client: &Client, table_name: &str, pk_name: &str, pk_len: usize) -> Self {
```

Comment:
Does clickhouse support multiple pk?

And what does `pk_len` means here? Do you mean we can delete multiple pk values?